src/billable-services/bill-waiver/utils.ts

```diff
@@ -1,28 +1,27 @@
 import { type OpenmrsResource } from '@openmrs/esm-framework';
-import type { LineItem, MappedBill } from '../../types';
-
-const WAIVER_UUID = 'eb6173cb-9678-4614-bbe1-0ccf7ed9d1d4';
+import type { BillableService, LineItem, MappedBill } from '../../types';
 
 export const createBillWaiverPayload = (
   bill: MappedBill,
   amountWaived: number,
   totalAmount: number,
   lineItems: Array<LineItem>,
-  billableLineItems: Array<OpenmrsResource>,
+  billableLineItems: Array<BillableService>,
 ) => {
   const { cashier } = bill;
+  const waiverUuid = findBillableServiceWaiverUuid(billableLineItems);
 
   const billPayment = {
     amount: parseFloat(totalAmount.toFixed(2)),
     amountTendered: parseFloat(Number(amountWaived).toFixed(2)),
     attributes: [],
-    instanceType: WAIVER_UUID,
+    instanceType: waiverUuid,
   };
 
   const processedLineItems = lineItems.map((lineItem) => ({
     ...lineItem,
     billableService: findBillableServiceUuid(billableLineItems, lineItem),
-    paymentStatus: 'PAID',
+    paymentStatus: 'PENDING',
```

Comment:
I think the wavier was meant to be a `complete payoff` to the bill & not a `discount`.